### DISCUSSION TITLE
Physics: gravitational lensing

### DISCUSSION THREAD

**Discussion by JamesMunday98** [phoebe-project/phoebe2#564] [Ideas]
Hello.

I can't find any mention of it in any release paper, the docs or any discussion - does phoebe include gravitational lensing in eclipsing systems? The impact is small, but still present.

The only place it has been spoken about is ancient https://sourceforge.net/p/phoebe/mailman/message/29015920/

Many Thanks,
James

**Comment by kecnry** [comment]
Hi James - No, unfortunately PHOEBE does not support gravitational lensing, but does support [gravitational redshift on RVs](http://phoebe-project.org/docs/2.3/tutorials/grav_redshift).  The observables are "collapsed" along the line of sight without any sort of ray tracing, so a full implementation of lensing would likely be impossible without a full redesign of the eclipsing logic, but an analytic approximation might be feasible.

Thanks,
Kyle

**Comment by JamesMunday98** [comment]
Thanks for getting back to me Kyle. Will bare that in mind.
